### ISSUE TITLE
Release to CRAN

### ISSUE THREAD

**Issue by ethanwhite** [weecology/ratdat#5]
This package hasn't been released to CRAN since it wasn't being actively used, but work is now happening to rewrite Data Carpentry lessons around it (see #4). 

@MCMaurer - would it be useful at this point to start working towards a CRAN release?

**Comment by MCMaurer** [comment]
@ethanwhite I think that would be great, it would definitely simplify the process for use in the Data Carpentry lessons. I'm happy to help with anything needed to make it happen. I've never done a CRAN-released package, but I'm happy to learn!

**Comment by ethanwhite** [comment]
Sounds good. I added automated testing, cleaned up the DESCRIPTION file, and did a bunch of cleanup based on feedback from win-builder, R-Hub, and R CMD check --as-cran. Take a look at #7 when you get a chance and we'll go from there.

**Comment by MCMaurer** [comment]
Hey @ethanwhite, my apologies for dropping the ball on this! I'm finally getting near the end of the work on the DC R Ecology lesson, and I realized I hadn't followed up on this. I have a few quick ideas to run by you:

- I think it might be worth changing the factors in all of the data.frames to characters, in order to reflect the more common starting point of character vectors since the update to `stringsAsFactors = F`.
- In the new R Ecology lesson, I've structured it so that learners start out working with ~50% of the data, only through the year 1989. Later in the lesson, the learners get messy versions of the surveys, species, and plots data, containing the later half of the data, from 1990 onwards. While I don't think we need to add those messy versions to the package, for the sake of the lesson it would be useful to have a version of the `complete.rda` that only has data through 1989. While it's obviously a more limited use case (only really useful for the DC lesson), it would make a big difference for that narrow use. If this is out of scope for the lesson, no worries!

I wanted to run these ideas by you first, and then I would be happy to open PRs to address both of them, and then take a look at getting it submitted to CRAN!

**Comment by ethanwhite** [comment]
Those both sound great. Go ahead and submit PRs when you're ready. I'm going to go ahead and merge in #7 now to avoid any conflicts as you make the additional changes.

**Comment by MCMaurer** [comment]
@ethanwhite thanks for merging those two PRs! At this point I think the data match up exactly with what I'm planning for the R Ecology lesson. Happy to help out with any other steps needed for a CRAN submission.

**Comment by ethanwhite** [comment]
Great! I've rerun everything through all of the checks again and everything looks good.

Would you like to be added as a contributor prior to submission? We generally believe in crediting all contributors. Same question for @aurielfournier & @fmichonneau who both changes in the history. If so please PR a change to the DESCRIPTION file. If not let me know so I know not to wait on submitting.

**Comment by MCMaurer** [comment]
Fantastic, just submitted a PR to be added to the contributor list!

**Comment by ethanwhite** [comment]
Submitted to CRAN this morning

**Comment by MCMaurer** [comment]
Fantastic, thanks Ethan!

**Comment by ethanwhite** [comment]
We are now up on CRAN!
